examples/fsharp.extraction.targets

```diff
@@ -87,7 +89,7 @@
         
         <!-- Run extraction for .fst files -->
         <!-- TODO: What about interfaces? Should we at least run verification for them? -->
-        <Exec ConsoleToMsBuild="true" Condition="'%(Compile.Extension)' == '.fst'" Command="$(FSTAR_EXE) $(FSTAR_WARN_ERROR) $(FSTAR_EXTRACTION_FLAGS) $(FSTAR_FLAGS) %(Compile.Identity) --extract &quot;+%(Compile.Filename)&quot;" IgnoreStandardErrorWarningFormat="true" />
+        <Exec ConsoleToMsBuild="true" Condition="'%(Compile.Extension)' == '.fst'" Command="$(CALL_PROC_EXE) $(FSTAR_EXE) $(FSTAR_WARN_ERROR) $(FSTAR_EXTRACTION_FLAGS) $(FSTAR_FLAGS) %(Compile.Identity) --extract &quot;+%(Compile.Filename)&quot;" IgnoreStandardErrorWarningFormat="true" />
```

Comment:
@mateuszbujalski @kant2002 

I retained a version of fsharp.extraction.targets, but wrapped it with CallProc, a small .NET program I wrote to wrap the call to a native fstar.exe. This works on both Windows & Linux

Reply:
If there's a way to make this work on Linux without the CallProc wrapper that would be great ... happy to have a PR for that